lms/static/js/spec/main.js

```diff
@@ -181,22 +181,7 @@
             },
             'underscore': {
                 deps: ['underscore.string'],
-                exports: '_',
-                init: function(UnderscoreString) {
```

Comment:
I was suggesting to move away from it in this work, if possible. Would that be too much to bite off? If so, let's create a techdebt ticket.